torchdata/nodes/loader.py

```diff
@@ -46,6 +46,7 @@ def __init__(
         self.loader = loader
         self.root = loader.root
         self._cached_item = None
+        self._saved_currect_state_dict: Optional[Dict[str, Any]] = None
```

Comment:
I purposely avoided `_cached_state_dict`, because _cached_state_dict is referring to the current state dict, and cached_item is referring to the next item. i.e. noth _cached_state_dict and cached_item do not correspond. wdyt

Reply:
Hmm, I see what you mean, but it could also be interpreted as "next call returns x"; ie there's nothing in the name that suggests it's for before or after the next item, just that we need to hold on to it